File: Java/FizzBuzz.java
public class FizzBuzz
{
	public static void main(String[] args) {
		StringBuilder builder = new StringBuilder();
		for(int i=1;i<=100;i++)
		{
		    if(i%3==0 && i%5==0)
		        builder.append("FizzBuzz, ");
		    else if(i%3==0)
		        builder.append("Fizz, ");
		    else if(i%5==0)
		        builder.append("Buzz, ");
		    else
		        builder.append(i + ", ");
		}
		builder.deleteCharAt(builder.length()-1);
		System.out.println(builder.toString());
	}
}

File: Haskell/fizzbuzz.hs
-- oonat
funcfizzbuzz:: Int -> String
funcfizzbuzz n 
  | n `mod` 15 == 0  = "FizzBuzz"
  | n `mod` 3  == 0  = "Fizz"
  | n `mod` 5  == 0  = "Buzz"
  | otherwise        = show n

main = print $ [ funcfizzbuzz x | x <-[1..100]]